Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:stretch
+FROM node:buster
 
 RUN apt-get update \
-    && apt-get install -y ruby2.3 bundler \
+    && apt-get install -y ruby2.5 bundler \
```

Comment:
Might be a good idea to go straight to 2.7 while you're at it. 2.7 is what we're running in most other images.

Reply:
I updated to 2.5 as an initial test to see how much broke. This is only on a `buster` image also.